### DISCUSSION TITLE
Add TikTok in Social links

### DISCUSSION THREAD

**Discussion by SeraphicRav** [withastro/starlight#2255] [Feature Requests]
### What version of `starlight` are you using?

0.26.1

### What is your idea?

There is a lot of social links provided but TikTok is not (there are no icon for it either).

### Why is this feature necessary?

TikTok is a big social network.

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by delucis** [comment]
Would be happy to merge a PR for this! Here’s a good example of a recent PR adding a social icon link: https://github.com/withastro/starlight/pull/2135

**Comment by SeraphicRav** [comment]
@delucis Hello ! Made a PR to add TikTok.
https://github.com/withastro/starlight/pull/2265

**Comment by HiDeoo** [comment]
Starting with Starlight [`v0.26.2`](https://github.com/withastro/starlight/releases/tag/%40astrojs%2Fstarlight%400.26.2), a TikTok social icon is now available thanks to @SeraphicRav 🙌
